lib/active_merchant/billing/gateways/quickbooks.rb

```diff
@@ -128,8 +128,8 @@ def scrub(transcript)
           gsub(%r((oauth_nonce=\")\w+), '\1[FILTERED]').
           gsub(%r((oauth_signature=\")[a-zA-Z%0-9]+), '\1[FILTERED]').
           gsub(%r((oauth_token=\")\w+), '\1[FILTERED]').
-          gsub(%r((number\D+)\d{16}), '\1[FILTERED]').
-          gsub(%r((cvc\D+)\d{3}), '\1[FILTERED]').
+          gsub(%r((number\D+)\d{13,19}), '\1[FILTERED]').
+          gsub(%r((cvc\D+)\d{3,4}), '\1[FILTERED]').
           gsub(%r((Authorization: Basic )\w+), '\1[FILTERED]').
```

Comment:
Why use a specified range length of characters instead of `d+`?

Reply:
Probably because that is what I wrote in the ticket. These should match specified ranges for these values currently, but `\d+` would keep us safe going forward, that's a good call.

Reply:
I've updated it to be` \d+`